recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module_1.9.bbappend

```diff
@@ -5,3 +5,6 @@ SRCREV = "${AUTOREV}"
 
 EXTRA_OEMAKE += "PVRSRV_VZ_NUM_OSID=${XT_PVR_NUM_OSID}"
 
+do_install_append() {
+    sed -i 's!platform/!!' ${D}/etc/udev/rules.d/72-pvr-seat.rules
```

Comment:
I was wrong. "soc" must be removed. The correct path is 
"/devices/platform/fd000000.gsx/drm/*"

Reply:
Done